docs/solidity-by-example.rst

```diff
@@ -620,4 +620,52 @@ Safe Remote Purchase
 Micropayment Channel
 ********************
 
-To be written.
+::
+
+    pragma solidity ^0.4.11;
+    contract Channel {
+        address[2] users;
+        uint deposit;
+        uint timeout;
+        uint closeTimestamp;
+        uint closeSequenceNumber;
+        uint closeBalanceUser0;
+        enum State { Opened, Accepted, Closing, Closed }
+        State state;
+        function Channel(address _other, uint _timeout) payable {
```

Comment:
I think some whitespaces between functions, state variables and the pragma cannot hurt.